bangazonapi/views/profile.py

```diff
@@ -330,7 +330,7 @@ class UserSerializer(serializers.HyperlinkedModelSerializer):
     """
     class Meta:
         model = User
-        fields = ('first_name', 'last_name', 'email')
+        fields = ('id', 'first_name', 'last_name', 'email')
```

Comment:
This code should not be changed, because the serializer has no effect on the data retrieved from your query above.